app-utils/unar/spec

```diff
@@ -1,5 +1,7 @@
-VER=1.10.1
-REL=6
-SRCS="tbl::http://http.debian.net/debian/pool/main/u/unar/unar_$VER.orig.tar.xz"
-CHKSUMS="sha256::963bbb25d0c321c9e022336318aacbaa3197e22063dd639c467453a4af8708ee"
+VER=1.10.8
+SRCS="git::commit=tags/v${VER};rename=XADMaster::https://github.com/MacPaw/XADMaster.git \
+      git::commit=tags/1.1;rename=UniversalDetector::https://github.com/MacPaw/universal-detector.git"
```

Comment:
This should be added back as

```
UPSTREAM_VER=1.0.1
UNIVERSALDETECTOR_VER=1.1
VER=${UPSTREAM_VER}+universaldetector${UNIVERSALDETECTOR_VER}
```

Reply:
Universal Detector didn't receive any tag since 2019 or any commits since 2022.